packages/graphql-dynamodb-transformer/src/DynamoDBModelTransformer.ts

```diff
@@ -31,6 +31,8 @@ import {
 import { ModelDirectiveArgs, getCreatedAtFieldName, getUpdatedAtFieldName } from './ModelDirectiveArgs';
 import { ResourceFactory } from './resources';
 
+const METADATA_KEY = 'DynamoDBTransformerMetadata';
```

Comment:
I like the use of metadata here, I was thinking of something different, but have 1 key per transformer in there and utilize key-value pairs for all values, even if just a boolean, but not depending only on the key's presence. It is up to you if you want to change it now, we can do that later as well.

Reply:
For now its internal to dynamodb transformer. If we are exposing this, we should also design for the ordering of these, as some of these when they get rendered in mapping template